CMakeLists.txt

```diff
@@ -170,10 +170,10 @@ INSTALL(FILES src/binaryen-c.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 # if binaryen.js and wasm.js were built (using "./build-js.sh", currently
 # optional), install them
-IF(EXISTS "bin/binaryen.js")
+IF(EXISTS "${CMAKE_SOURCE_DIR}/bin/binaryen.js")
   INSTALL(FILES bin/binaryen.js DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
```

Comment:
hmm, don't we also need the prefix for `bin/binaryen.js` on this line as well (and below)?